template/node12/index.js

```diff
@@ -85,9 +85,9 @@ class FunctionContext {
     }
 
     fail(value) {
```

Comment:
Think of this example, it will return a HTTP 200, and I don't think that's what you wanted.

```js
  return context
    .fail(result)
```

The code should probably stay the same as it is, but then change the middleware closure, with something like this:

```js
            return res.status(fnContext.status() || 500)
```

Reply:
Can you share a handler example that we can use for an end to end test?

```js
return context
    .fail("Not OK")
```

Should give 500 and "Not OK"

```js
return context
    .succeed("OK")
```

Gives 200 and "OK"

```js
return context.status(401)
    .fail("Not authorized")
```

Gives 401 - Not authorized